gamemodes/terrortown/gamemode/roles/medium/cl_medium.lua

```diff
@@ -108,6 +120,184 @@ hook.Add("Think", "Medium_RoleFeature_Think", function()
     end
 end)
 
+---------------
+-- SPIRIT ID --
+---------------
+
+local roleback = surface.GetTextureID("vgui/ttt/sprite_roleback")
+local rolefront = surface.GetTextureID("vgui/ttt/sprite_rolefront")
+
+hook.Add( "PostDrawTranslucentRenderables", "Medium_PostDrawTranslucentRenderables", function()
+    if medium_seance_max_info:GetInt() == MEDIUM_SCANNED_NONE then return end
+
+    local client = LocalPlayer()
+    if not client:IsActiveMedium() then return end
```

Comment:
I've seen timings where `LocalPlayer` isn't set yet, so should probably add that
```suggestion
    if not IsPlayer(client) or not client:IsActiveMedium() then return end
```